deploy-manage/toc.yml

```
@@ -601,7 +601,8 @@ toc:
               - file: users-roles/cluster-or-deployment-auth/manage-authentication-for-multiple-clusters.md
           - file: users-roles/cluster-or-deployment-auth/user-roles.md
             children:
-              - file: users-roles/cluster-or-deployment-auth/built-in-roles.md
+              - title: "Roles"
```

to differentiate from creating your own roles

```suggestion
              - title: "Built-in roles"
```

---

Thanks, @shainaraskas , that's a good point! 

Fabrizio was wondering if we should somehow indicate to users that we're going to send them to the Reference section. Something like this maybe? Wdyt?
```suggestion
  - title: "Built-in roles (Reference)"
```